docs/physics/index.rst

```diff
@@ -0,0 +1,28 @@
+=========
+pace-physics
+=========
+
+pace-physics includes the Python implementation of GFS physics built using the GT4Py domain-specific language.
+Currently, only GFDL cloud microphysics is integrated into Pace.
+Additional physics schemes (NOAH land surface, GFS sea ice, scale-aware mass-flux shallow convection, hybrid eddy-diffusivity mass-flux PBL and free atmospheric turbulence, and rapid radiative transfer model)
+have been ported indendepently and are available in the `physics-standalone`_ repository,
+additional work is required to integrate these schemes.
+
+State container is highly recommended to keep track of physics variables
+because only certain variables interact with the rest of the model.
+This also makes it easier to follow variables shared across physics schemes in the future.
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   state
+   api
+
+Indices and tables
+==================
```

Comment:
```suggestion
------------------
```
I think this is necessary to avoid this section appearing int he table of contents at the same level as e.g. "Microphysics".